### DISCUSSION TITLE
Increase landing page lead copy contrast

### DISCUSSION THREAD

**Discussion by bolatovumar** [btcpayserver/btcpayserver#2440] [Ideas / Feature Requests]
Currently the lead copy on the landing page uses light weight which makes it appear low-contrast and hard to read IMO. Especially if you compare it with other elements on the page which are high-contrast.

![Screen Shot 2021-04-06 at 6 34 13 PM](https://user-images.githubusercontent.com/1934678/113803567-a5648a80-9711-11eb-9f60-38a6cdfe72ba.png)

If you bump up the font size to 500 it's more contrast-y:

![Screen Shot 2021-04-06 at 6 34 22 PM](https://user-images.githubusercontent.com/1934678/113803573-a72e4e00-9711-11eb-8ebb-c30f13d54487.png)

Even more contrast-y at 600:

![Screen Shot 2021-04-06 at 6 34 32 PM](https://user-images.githubusercontent.com/1934678/113803577-a85f7b00-9711-11eb-8e25-26e41bd36ed9.png)

Paging in @dstrukt.

**Comment by Zaxounette** [comment]
ACK.
The 600 contrast is good IMO. Gives kind of a "bold" effect, making it even more readable.

**Comment by pavlenex** [comment]
Just a note, this landing page will eventually be obsoleted by a #1711 but it'll take some time for that.

Concept ACK since it's a minor change.
